### DISCUSSION TITLE
Unable to add ZY-M100 WiFi Presence Sensor

### DISCUSSION THREAD

**Discussion by aatifk** [make-all/tuya-local#1276] [Q&A]
I'm trying to add the ZY-M100 WiFi version presence sensor using Local Tuya. I can see the device is supported, and I have the device ID and local key. Adding the device however, results in an error that says it can't be added with these details. Not sure what to do here. The sensor works OK with the standard cloud-based Tuya integration (which is what I'm trying to move away from). I have three other devices running fine via Local Tuya (two lightbulbs and a XiuMii sensor). What am I doing wrong?

**Comment by make-all** [comment]
How is the device powered?

**Reply by aatifk** [reply]
They're mains powered. I believe they're pretty common now, come in both Zigbee and WiFi variants.

**Comment by make-all** [comment]
See #522 for ideas on resolving connection issues.
